### PR TITLE
dates in new line

### PR DESCRIPTION
**Pull request by Arnab-Developer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,7 @@
 date
 
 10 May 2024 10:30 am
+
 update 25-may-2024
+
 23 June 2024
```
Comment (arnab-jit1) [inline README.md]: Nice work

File: README.md [hunk 2]
```diff
@@ -2,5 +2,9 @@
 date
 
 10 May 2024 10:30 am
+
 update 25-may-2024
-23 June 2024
+
+23-June-2024
```
Comment (arnab-jit1) [inline README.md:8]: add time

File: README.md [hunk 3]
```diff
@@ -2,5 +2,9 @@
...
+
+04-July-2024 10:20 am
```
Comment (arnab-jit1) [inline README.md:10]:
```suggestion
04-July-2024 10 hh 20 mm AM
```

**Review by arnab-jit1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Arnab-Developer) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -2,5 +2,9 @@
 date
 
 10 May 2024 10:30 am
+
 update 25-may-2024
-23 June 2024
+
+23-June-2024 05 hh 32 mm pm
```
Comment (arnab-jit1) [inline README.md:8]:
```suggestion
23-June-2024 05 hh 32 mm PM
```

**Review by arnab-jit1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Arnab-Developer) [reply README.md]: done

**Review by arnab-jit1 — APPROVED** [review]

**Pull request merged**